website/docs/sdk-reference/python.md

```diff
@@ -488,13 +488,23 @@ client = configcatclient.get(
 
 ## Logging
 
-In the _ConfigCat SDK_, a default logger writes logs to the standard output. The following example shows how to set the _Log Level_ of the default logger.
+The _ConfigCat SDK_ uses Python's built-in <a href="https://docs.python.org/3/library/logging.html" target="_blank">logging library</a>. 
+The default logger writes logs to the standard output. The following example shows how to set the _Log Level_ of the default logger.
 
 ```python
 import logging
 logging.basicConfig(level=logging.INFO)
 ```
 
+The following example shows how to set the _Log Level_ only on the internal _ConfigCat_ logger.
```

Comment:
What's the difference between this and the above example?

Reply:
In python there is a default (root) logger. The first example sets the log level of that logger. In this case all logger will use this log level (the inner configcat logger too).

The second example sets the log level of the inner (configcat) logger.  

There was a question about this from a customer. They want to hide just the configcat logs.

Reply:
Yeah, what I meant is to please explain this to the reader.